lib/xml_security.rb

```diff
@@ -238,7 +249,7 @@ def validate_signature(base64_cert, soft = true)
       REXML::XPath.each(@sig_element, "//ds:Reference", {"ds"=>DSIG}) do |ref|
         uri                           = ref.attributes.get_attribute("URI").value
 
```

Comment:
In the onelogin pull request, I see the following block in `validate_signature`, but not in this PR. Was that omitted intentionally?
```ruby
document = Nokogiri.parse(self.to_s) do |options|
  options = XMLSecurity::BaseDocument::NOKOGIRI_OPTIONS
end
```